docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -55,175 +58,154 @@ to start the daemon.
+
+    * Use a silicon crystal structure
+    * Define a ``2x2x2`` mesh of k-points. 
+      
+  Note: If you just copy and paste code that you executed previously, 
+  this may result in duplication of information on your database.
+  In fact, you can re-use an existing structure stored in your database [#f1]_. Use a combination of the bash command
+  ``verdi data structure list`` and of the shell command ``load_node()``
+  to get an object representing the structure created earlier.
 
 Attaching the input information to the calculation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-So far we have defined (or loaded) some of the input data, but we
-haven’t instructed the calculation to use them. To do this, let’s just
-set the appropriate attributes of the builder (we assume here that you
-created the structure and k-points AiiDA nodes before and called them
-``structure`` and ``kpoints``, respectively):
+Once you've created a ``structure`` node and a ``kpoints`` node,
+attach it to the calculation:
 
 .. code:: python
 
     builder.structure = structure
     builder.kpoints = kpoints
 
-Note that you can set in the builder both stored and unstored nodes.
-AiiDA will take care of storing the unstored nodes upon submission.
-Otherwise, if you decide not to submit, nothing will be stored in the
-database.
+.. note:: 
 
-Moreover, PWscf also needs information on the pseudopotentials,
-specified by UpfData objects. This is set by storing a dictionary in
-“builder.pseudo”, with keys being the kind names, and value being the
-UpfData pseudopotential nodes. To simplify the task of choosing
-pseudopotentials, we can however use a helper function that
-automatically returns this dictionary picking the pseudopotentials from
-a given UPF family.
+  The builder accepts both *stored* and *unstored* data nodes.
+  AiiDA will take care of storing the unstored nodes upon submission.
+  If you decide not to submit, nothing will be stored in the database.
 
+PWscf also needs information on the pseudopotentials,
+in the form of a dictionary, where keys are the names of the elements and the values are the corresponding ``UpfData`` objects containing the information on the pseudopotential.
+However, instead of creating the dictionary by hand, we can use a helper function that picks the right pseudopotentials for our structure from a pseudopotential *family*.
 You can list the preconfigured families from the command line:
 
 .. code:: console
 
     verdi data upf listfamilies
 
-Pick the one you configured earlier or one of the ``SSSP`` families that
-we provide, and link it to the calculation using the command:
+Pick the one you configured earlier (or one of the ``SSSP`` families that
+we provide) and link it to the calculation using the command:
 
 .. code:: python
 
     from aiida.orm.nodes.data.upf import get_pseudos_from_structure
     builder.pseudos = get_pseudos_from_structure(structure, '<PSEUDO_FAMILY_NAME>')
 
```

Comment:
Maybe give a hint to print the `builder` or even `builder.pseudos` to see the contents of this namespace after the call